src/lib/Sympa/Message.pm

```diff
@@ -2145,6 +2162,15 @@ sub _urlize_one_part {
         $filename = Encode::encode_utf8($filename)
             if Encode::is_utf8($filename);
     } else {
+        my $content_disposition = $entity->head->get('Content-Disposition');
```

Comment:
Use ``mime_attr()`` to handle things like "``Content-Disposition: (comment) Attachment (comment)``":
```perl
         my $content_disposition = lc($entity->head->mime_attr('Content-Disposition') // '');
```
and no regexp is required to compare it with ``'attachment'``.

Reply:
Sure? No spaces after/before the field content?
In that case, I'll do as you suggest.
Regards,

Reply:
get() gets value of the field, ie. “ `(comment) Attachment (comment); filename="name.ext"`”. mime_attr() gets specified attribute in the field, ie. “ `Attachment `”

Reply:
Thanks @ikedas !
I really need to dig deeper into MIME::Tools.
Fixed in commit 72451a0.
Cheers,